### ISSUE TITLE
PHP not highlighting syntax

### ISSUE THREAD

**Issue by wraytw** [conorhastings/react-syntax-highlighter#114]
I am trying to get php code to highlight on my webpage, but the code is not being syntactically highlighted. Below is my entire react app: 

```javascript
import React from 'react'
import ReactDOM from 'react-dom'
import Syntax from 'react-syntax-highlighter/prism'

const phpCode = `
$data = array(
  'name' => 'James Brown',
  'email' => 'jbrown45@example.com',
  'phone' => '224-555-6798'
);
$data_string = json_encode($data);

$req = curl_init('/users');
curl_setopt($req, CURLOPT_CUSTOMREQUEST, 'POST');
curl_setopt($ch, CURLOPT_POSTFIELDS, $data_string);
curl_setopt($req, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_HTTPHEADER, array(
    'Content-Type: application/json',
    'Content-Length: ' . strlen($data_string))
);

$res = curl_exec($req);
print($res);
`

ReactDOM.render(<Syntax language="php">{phpCode}</Syntax>, document.getElementById('root'))
```

That does not highlight the php code.

I'm using:
- React 16.2.0
- react-syntax-highlighter 7.0.2

Thanks

**Comment by conorhastings** [comment]
thanks for providing the details,  I'll take a look this weekend, if you have a screenshot that would also be super helpful

**Comment by wraytw** [comment]
I’ll do ya one better, here’s the site https://userapi.io/docs

And here’s the code https://github.com/wraytw/userapi.io-website/blob/master/src/shared/CodeBlock.js

**Comment by conorhastings** [comment]
sorry for the late response, @wraytw I am looking into this, just busy with a new job

**Comment by conorhastings** [comment]
it seems to be a problem with the php highlighting specifically, see small reproduction here -- https://codesandbox.io/s/kxzpnqv5l7 (look at src/index.js) any ideas @wooorm

**Comment by wooorm** [comment]
It seems to be the order of things: https://github.com/wooorm/refractor/blob/0f04ee551b9030e275a9e3ee35c656f9750b36f3/index.js#L102-L103

Yes, you’re getting this live :p

**Comment by wooorm** [comment]
I think I have it!

**Comment by conorhastings** [comment]
fixed in `v7.0.4` @wraytw , thanks @wooorm !
